packages/mermaid/src/docs/.vitepress/config.ts

```diff
@@ -101,14 +106,11 @@ function sidebarAll() {
   return [
     {
       text: '📔 Introduction',
-      collapsible: true,
+      collapsed: true,
```

Comment:
Why collapse everything?

Reply:
This option did not worked at all, I simply renamed the option, the value left intact. If we want our sidebar to be expanded by default, I'll change its value too

Reply:
No everything is expanded by default and menu is collapsible. If we don't want to make it collapsible, we need to remove the option entirely